docs/background_concepts.ipynb

```diff
@@ -1,970 +0,0 @@
-{
```

Comment:
Is deleting this intentional or is this content now somewhere else?

Reply:
This is now the floirs_101 notebook with the same concept. I changed the name, to try to move towards something more direct because `background_concepts` and the matching title don't scream at me to check this out if I'm new. I'm not wed to the floris_101, so we can definitely change that too.

Reply:
I think maybe you didn't add that notebook, I don't see it in the changed files.

Reply:
Just checking in on this - is there meant to be a notebook called `floris_101.ipynb`?